docs/content/reference/metadata.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "MMv1 metadata reference"
+weight: 35
+---
+
+# MMv1 metadata reference
+
+This page documents all properties for metadata.
```

Comment:
Could you add information about when this file is needed? You mention in you PR that most resources would be unaffected. When would they be affected?

Reply:
I just added some clarification. The file is autogenerated for MMv1, so it is only really needed for handwritten resources (which is why that guide was updated as well).